addon/components/o-s-s/checkbox.ts

```diff
@@ -43,6 +44,16 @@ export default class OSSCheckbox extends Component<OSSCheckboxArgs> {
     return classes.join(' ');
   }
 
+  get fakeCheckboxClasses(): string {
+    const classes = ['upf-checkbox__fake-checkbox'];
+
+    if (this.args.partial) {
+      classes.push('upf-checkbox__fake-checkbox--partial');
```

Comment:
partial prevails over checked right ?
maybe add a test "If checked is truthy and partial is truthy, the checkbox is partially checked" ?